src/pybind/cephfs/cephfs.pyx

```diff
@@ -500,12 +505,12 @@ cdef class LibCephFS(object):
             raise Error("libcephfs_initialize failed with error code: %d" % ret)
 
         self.state = "configuring"
-        if conffile is not None:
-            # read the default conf file when '' is given
-            if conffile == '':
-                conffile = None
-            self.conf_read_file(conffile)
-        if conf is not None:
+        # skip reading default conf
+        if conffile is False:
+            pass
+        else:
+            self.conf_read_file(None if conffile == '' else conffile)
```

Comment:
Do we want to maintain the backward compatibility of supporting reading no conf file when empty string is passed?

Reply:
Effectively it won't matter but it's more appropriate to move this code to [`conf_read_file()`](https://github.com/ceph/ceph/blob/master/src/pybind/cephfs/cephfs.pyx#L566). @gregsfortytwo Any opinion?